tests/prague/eip7685_general_purpose_el_requests/test_deposits_withdrawals_consolidations.py

```diff
@@ -468,6 +468,30 @@ def invalid_requests_block_combinations(fork: Fork) -> List[Any]:
             id="extra_invalid_type_request",
```

Comment:
I would just modify this test id to be more explicit and to better match the test cases removed from `tests/prague/eip7685_general_purpose_el_requests/test_request_types.py`
```suggestion
            id="extra_invalid_type_request_with_no_data",
```

Reply:
Makes total sense, thanks!